<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiform</class>
 <widget class="QMainWindow" name="uiform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>642</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Course 3</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#f3f3f3;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>290</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 royalblue, stop:1 skyblue);border-radius:7px;</string>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>241</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;color:white;font-weight:bold;border:1px solid white;</string>
       </property>
       <property name="title">
        <string>x0</string>
       </property>
       <widget class="QLineEdit" name="xs_edit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;color:black;border-radius:3px;</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>79</y>
         <width>241</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;color:white;font-weight:bold;border:1px solid white;</string>
       </property>
       <property name="title">
        <string>xn</string>
       </property>
       <widget class="QLineEdit" name="xn_edit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;color:black;border-radius:3px;</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>241</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;color:white;font-weight:bold;border:1px solid white;</string>
       </property>
       <property name="title">
        <string>Points</string>
       </property>
       <widget class="QLineEdit" name="points_edit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;color:black;border-radius:3px;</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>269</x>
         <y>190</y>
         <width>241</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;color:white;font-weight:bold;border:1px solid white;</string>
       </property>
       <property name="title">
        <string>Order</string>
       </property>
       <widget class="QLineEdit" name="order_edit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>19</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;color:black;border-radius:3px;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>501</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;color:white;font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Draw:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="function_box">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background:white;color:black;padding:7px;border-radius:4px;height:20px;width:100%!important;</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Sine integral</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cosine integral</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bessel function</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fresnel integral S</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fresnel integral C</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:7px;border:2px solid gainsboro;background:white;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="log_txt_browser">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;border-radius:7px;</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="c_Btn">
        <property name="styleSheet">
         <string notr="true">border:0;background:royalblue;color:white;border-radius:4px;padding:10px;</string>
        </property>
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="com_Btn">
        <property name="styleSheet">
         <string notr="true">border:0;background:royalblue;color:white;border-radius:4px;padding:10px;</string>
        </property>
        <property name="text">
         <string>compute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d_btn">
        <property name="styleSheet">
         <string notr="true">border:0;background:royalblue;color:white;border-radius:4px;padding:10px;</string>
        </property>
        <property name="text">
         <string>disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="e_btn">
        <property name="styleSheet">
         <string notr="true">border:0;background:royalblue;color:white;border-radius:4px;padding:10px;</string>
        </property>
        <property name="text">
         <string>exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
